content/en/docs/ops/deployment/multicluster-considerations/index.md

```diff
@@ -0,0 +1,155 @@
+load balancing for these services.
+
+{{< image width="80%"
+    link="endpoint-discovery.svg"
+    caption="Primary clusters with endpoint discovery"
+    >}}
+
+By default, Istio will load balance requests evenly between endpoints in each cluster. In large systems that span
+geographic regions, it may be desirable to use locality load balancing to prefer that traffic stay in the same zone
+or region.
+
+In some advanced scenarios, load balancing across clusters may not be desired. For example, in a blue/green
+deployment, you may deploy different versions of the system to different clusters. In this case, each cluster is
+effectively operating as an independent mesh. This behavior can be achieved in a couple of ways:
+
+* Do not exchange remote secrets between the clusters. This offers the strongest isolation between the clusters.
+* Use `VirtualService` and `DestinationRule` to disallow routing between two versions of the services.
+
+In either case, cross-cluster load balancing is prevented. External traffic can be routed to one cluster or the
+other using an external load balancer.
+
+{{< image width="80%"
+    link="blue-green.svg"
+    caption="Blue-green deployment without cross-cluster load balancing"
+    >}}
+
+### Remote Clusters
+
+A {{< gloss >}}remote cluster{{< /gloss >}} refers to a cluster that contains user workloads but no control plane.
+Remote clusters rely on a control plane residing either in a primary cluster or
+{{< gloss "external control plane" >}}external to the mesh{{< /gloss >}}. In either case, the control plane must be
+made accessible to the remote cluster.
+
+Cloud vendors may provide options, such as internal load balancers, for providing this sort of capability
+without exposing the control plane on the public internet. Such an option, if it exists and meets your requirements,
+will likely be the best choice.
+
+{{< image width="80%"
+    link="remote-clusters.svg"
+    caption="Remote clusters"
+    >}}
+
+## DNS and Service Deployment
+
+When a client application makes a request to some host, it must first perform a DNS lookup for the hostname to
+obtain an IP address before it can proceed with the request.
+
+In Kubernetes, the DNS server residing within the cluster typically handles this DNS lookup. Kubernetes
+services configure this server.
+
+Istio ignores the IP address returned from the DNS lookup, however. Instead, it
+uses the list of active endpoints for the host and load balances across them. Istio uses either Kubernetes
+`Service`/`Endpoint` or Istio `ServiceEntry` to configure its internal mapping of hostname to IP addresses.
+
+This two-tiered naming system becomes more complicated when you have multiple clusters. Istio is inherently
+multicluster-aware, but Kubernetes is not (today). Because of this, for a client in one cluster to make a request to
+a service in another cluster, there must be a Kubernetes service for the host defined in the client cluster in order
```

Comment:
```suggestion
a service in another cluster, there must be a Kubernetes `Service` for the host defined in the client cluster in order
```

Reply:
I've added some clarification to the DNS section regarding `ServiceEntry`.